examples/install_requirements.py

```diff
@@ -45,7 +46,29 @@ def hasWhitespace(string):
 # Check if in virtual environment
 in_venv = getattr(sys, "real_prefix", getattr(sys, "base_prefix", sys.prefix)) != sys.prefix
 pip_call = [sys.executable, "-m", "pip"]
+pip_installed = True
 pip_install = pip_call + ["install"]
+
+try:
+    subprocess.check_call(pip_call + ["--version"])
+except subprocess.CalledProcessError as ex:
+    pip_installed = False
+
+if not pip_installed:
+    err_str = "Issues with \"pip\" package detected! Follow the official instructions to install - https://pip.pypa.io/en/stable/installation/"
+    raise RuntimeError(err_str)
+
+if sys.version_info[0] != 3:
+    raise RuntimeError("Demo script requires Python 3 to run (detected: Python {})".format(sys.version_info[0]))
```

Comment:
Modify the error message to be in context of this repo examples instead of depthai demo